src/components/EmptyState/EmptyState.scss

```diff
@@ -70,14 +69,66 @@ $illustration-width-cropped: calc(
   }
 }
 
-.Details {
-  position: relative;
-  z-index: z-index(details, $stacking-order);
-  padding: 0 spacing();
-  width: $detail-width;
+.withinContentContainer {
+  .Section {
+    position: unset;
```

Comment:
do you need unset?

Reply:
Yes, because the position is set to relative, when not in a content context, the position has to be unset or it still has a relative position which pushes it to the right a noticeable amount.